content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,120 @@
+
+### IPv4/IPv6 dual-stack support
+IP addresses are a consumable resource operators and cluster administrators need to make sure are not exhausted. Having dual-stack support enables native IPv6 routing to pods and services, and it also improves a possible scaling limitation for workloads.
+
+Dual-stack support in Kubernetes means pods, services, and nodes can get IPv4 addresses and IPv6 addresses. In Kubernetes 1.21 dual-stack support is graduating from alpha to beta and is now enabled by default.
+
+### Immutable Secrets and ConfigMaps
+Immutable Secrets and ConfigMaps adds a new field to those resource types which will reject changes to those objects if set. Secrets and ConfigMaps by default are mutable which is beneficial for pods that are able to consume changes. Mutating Secrets and ConfigMaps can also cause problems if a bad configuration is pushed for pods that use them.
+
+By marking Secrets and ConfigMaps as immutable you can be sure your application configuration won't change. If you want to make changes you'll need to create a new, uniquly named Secret or ConfigMap and deploy a new pod to consume that resource. Immutable resources also have scaling benefits because controllers do not need to poll the API server to watch for changes.
+This feature is now graduating to stable in Kubernetes 1.21.
+
+### Graceful Node Shutdown
+Graceful Node shutdown is graduating to beta with this release (and will now be available to a much larger group of users!). This is a hugely beneficial feature that allows Kubelet to be aware of Node shutdown, and gracefully terminate Pods that are scheduled to that Node.
+
+Currently, when a node shuts down, Pods do not follow the expected termination lifecycle and are not shutdown gracefully. This can introduce problems with a lot of different workloads. Going forward, Kubelet will be able to propagate the shutdown signal to Pods ensuring they can terminate as gracefully as possible.
+
+### PV Health Monitor
+Physical Volumes (PV) are commonly used in applications to get local, file-based storage. They can be used in many different ways and help users migrate applications without needing to re-write storage backends.
+
+Kubernetes 1.21 has a new alpha feature which allows PVs to be monitored for health of the volume and marked accordingly if the volume becomes unhealthy. Workloads will be able to react to the health state to protect data from being written or read from a volume that is unhealthy.
+
+### Reducing Kubernetes Build Maintenance
+Previously Kubernetes has maintained multiple build systems. This has often been a source of friction and complexity for new and current contributors.
+
+Over the last release cycle, a lot of work has been put in to simplify the build process, and standardize on the native Golang build tools. This should empower broader commmunity maintenance, and lower the barier to entry for new contributors.
+
+## Major Changes
+
+### PodSecurityPolicy Deprecation
```

Comment:
Should we mention that PodSecurityPolicy was previously beta? Some people might assume it was stable. The live docs won't show that it went from beta to deprecated (they'll have it marked as deprecated, which is correct!) - you'd have to look at the feature gate table.

Reply:
Looks good to me, thanks!